microsoft/testsuites/cloud_hypervisor/ch_tests.py

```diff
@@ -96,6 +96,31 @@ def verify_cloud_hypervisor_live_migration_tests(
             result, environment, "integration-live-migration", hypervisor
         )
 
+    @TestCaseMetadata(
```

Comment:
The changes of cloud hypervisor test cases should be in another PR, so it's easier to review and clearer history.

Reply:
+1

Reply:
Removed CH perf metric changes